WindowsServerDocs/identity/ad-fs/operations/Configure-AD-FS-to-authenticate-users-stored-in-LDAP-directories.md

```diff
@@ -11,7 +11,7 @@ ms.prod: windows-server
 
 ms.technology: identity-adfs
 ---
-# Configure AD FS to authenticate users stored in LDAP directories
+# Configure AD FS to authenticate users stored in LDAP directories in Windows Server 2016,2019.(2012R2 is not supported)
```

Comment:
just a simple suggestion of style & readability (spacing) improvement:
```suggestion
# Configure AD FS to authenticate users stored in LDAP directories in Windows Server 2016 and 2019 (2012 R2 is not supported)
```
Note that this is a header text (only 1 # in front), so a period character is not expected.